src/Tribe/Attendees.php

```diff
@@ -661,9 +661,18 @@ public function maybe_generate_csv() {
 			// Create the file pointer connected to the output stream.
 			$output = fopen( 'php://output', 'w' );
 
+			/**
+			 * Allow for modifying the field delimiter in the CSV export file.
+			 *
+			 * @since 5.1.2
+			 *
+			 * @param string delimiter The separator
+			 */
```

Comment:
Let's document the return as string.

Reply:
Filters don't have @return for phpdoc purposes